package.json

```diff
@@ -1,16 +1,24 @@
 {
   "name": "swl-builder",
+  "homepage": "https://lukezors.github.io/swl-builder",
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "enzyme": "^3.10.0",
+    "enzyme-adapter-react-16": "^1.15.1",
+    "gh-pages": "^2.1.1",
+    "prop-types": "^15.7.2",
     "react": "^16.11.0",
     "react-dom": "^16.11.0",
-    "react-scripts": "3.2.0"
+    "react-scripts": "3.2.0",
+    "styled-components": "^4.4.1"
   },
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "test": "react-scripts test",
+    "test": "react-scripts test --watchAll=false",
+    "predeploy": "yarn run build",
+    "deploy": "gh-pages -d build",
```

Comment:
Also if you're deploying manually, if you want to you can explore deploying to Github Pages through Travis CI on every commit - I believe it's free for public projects! Might sidetrack you from learning frontend though 😅

Reply:
I really want that stuff so will defo have to look into it, would be nice if it was built into github haha

Reply:
It pretty much is! Same company